build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
Chromium just announced support for completely headless runs, so it might actually be possible to run tests in a "browser" without spawning the whole app.